recipes/professor/meta.yaml

```
@@ -0,0 +1,85 @@
+{% set name = "Professor" %}
+{% set version = "2.3.4" %}
+{% set tag = name|lower ~ "-" ~ version %}
```

> the trick is to define the version in terms of the tag rather than the opposite

@chrisburr like this?

```suggestion
{% set tag = "professor-2.3.4" %}
{% set version = tag | replace("professor-", "") %}
```